frontend/src/app/forms/steps/field-designation-page/field-designation-page.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="main-description">
+  <h1>Industries and Positions</h1>
```

Comment:
What do you think - should we maybe use the same title as in the right menu? Otherwise, it might be confusing to the user.

Reply:
This is a good idea, but i was not sure if "field and designation" or "Industries and Position" is the better name and when we translate this later in german, because i thought this app should be in german, how will we name it then?

Reply:
Hm I like "Industries and Position" or "Industries and Positions" (you can select more than one position, hence the plural). 

Regadring the German version: How about directly translating it to "Industrien und Positionen" or sth like that?